model/vcs/registry.yaml

```diff
@@ -12,11 +12,32 @@ groups:
           The [URL](https://wikipedia.org/wiki/URL) of the repository
           providing the complete address in order to locate and identify the
           repository.
+        note: |
+          The URL SHOULD only be an HTTPS absolute URL and SHOULD NOT include
+          `.git` appended on the end nor be an SSH URL.
         examples:
           [
             "https://github.com/opentelemetry/open-telemetry-collector-contrib",
             "https://gitlab.com/my-org/my-project/my-projects-project/repo",
           ]
+      - id: vcs.repository.name
```

Comment:
Should we add this attribute as an optional attribute for any VCS metric which have a `vcs.repository.url.full` attribute?

Reply:
Yup, I think so. Wasn't planning on changing the metrics though here. Was only tackling the registry to keep things small.

Reply:
It's ok to follow up, but also referencing the attribute in this PR would be quite a small change

Reply:
I've went ahead and adjusted the metrics to include it as recommended.